kitsune/sumo/jinja2/base.html

```diff
@@ -58,6 +56,8 @@
   {% if canonical_url %}
     <link rel="canonical" href="{{ canonical_url|safe }}" />
   {% endif %}
+
+  {% include "includes/google-analytics.html" %}
```

Comment:
why is this moved?

Reply:
Artifact from working on a GA issue with @escattone - unrelated to this work.  Fixed!